test/dataframe.jl

```diff
@@ -582,4 +582,32 @@ module TestDataFrame
         df[4] = [1,2,3]
         @test names(df) == [:x3, :x3_1, :x3_2, :x4]
     end
+
+    @testset "handling of end in indexing" begin
+        z = DataFrames.DataFrame(rand(4,5))
```

Comment:
Why the `DataFrames` prefix? Same below.

Otherwise, looks good.

Reply:
leftover from initial testing. fixed.